### DISCUSSION TITLE
Embbed HTML file into a page

### DISCUSSION THREAD

**Discussion by arturocarriongh** [requarks/wiki#6984] [Help / Questions] [answered]
I was researching on almost each option to insert assests without success this situation. I have a HTML file that I need to embbed into a Wiki page. I upload the asset and I insert it in my page, but instead of rendering the HTML file, it creates a link that takes you to an non-existent page.  Any help on this? Thanks!

**Comment by NGPixel** [comment, marked as answer]
You can't have an HTML file as an asset to embed. Instead create a new page using the Raw code editor and paste the contents of the HTML file.

**Reply by arturocarriongh** [reply]
Hi Nicolas
I tried as you suggested by inserting markup of my file inside a raw HTML page but unfortunately it does not work. I copied the markup in between body tags of my file and pasted it inside the Wiki HTML raw page and notihing is rendered. I attached the sample HTML file I used. Let me know if I am doing something wrong, please.
[Sample_file.txt](https://github.com/requarks/wiki/files/14251561/Sample_file.txt)

**Comment by arturocarriongh** [comment]
Hi Nicolas
Thank you for your quick response!
I tried this workaround but it neither worked. The HTML file is a file produced by saving a Draw.io graph in HTML format, so it is rendering a diagram with a lot of shapes and arrows relating each. I am not authorized to send the original file since it has sensitive information about our company, but let me create a sample HTML file with draw.io and give it a try. I'll let you know how it`s worked. If I get same result I will send the out to you.

Thanks!
Greetings!


Arturo Carrion | Senior Software Developer

   [https://www.acceptallc.com/signature/cellphone.png] <tel:787-774-1555>          [https://www.acceptallc.com/signature/phone.png] <tel:+1787-774-1555>         [https://www.acceptallc.com/signature/mail.png] ***@***.***>         [https://www.acceptallc.com/signature/web.png] <https://www.acceptapayments.com/>
             [https://www.acceptallc.com/signature/Accepta_Earth.gif]

________________________________

[https://www.acceptallc.com/signature/facebook.png]<https://www.facebook.com/acceptapayments>

[https://www.acceptallc.com/signature/twitter.png]<https://twitter.com/acceptapayments>

[https://www.acceptallc.com/signature/linkedin.png]<https://www.linkedin.com/company/10634374/>

[https://www.acceptallc.com/signature/youtube.png]<https://www.youtube.com/channel/UC-68TI9KBwCioeSOSln_O7A>

________________________________

**Comment by arturocarriongh** [comment]
Hi again Nicolas
I tried with the attached file with the same results. The HTLM markup I embedded in the Wiki page is that in between the <body></body> tags from the file. Let me know if I am doing something wrong.
Thanks a lot!


Arturo Carrion | Senior Software Developer

   [https://www.acceptallc.com/signature/cellphone.png] <tel:787-774-1555>          [https://www.acceptallc.com/signature/phone.png] <tel:+1787-774-1555>         [https://www.acceptallc.com/signature/mail.png] ***@***.***>         [https://www.acceptallc.com/signature/web.png] <https://www.acceptapayments.com/>
             [https://www.acceptallc.com/signature/Accepta_Earth.gif]

________________________________

[https://www.acceptallc.com/signature/facebook.png]<https://www.facebook.com/acceptapayments>

[https://www.acceptallc.com/signature/twitter.png]<https://twitter.com/acceptapayments>

[https://www.acceptallc.com/signature/linkedin.png]<https://www.linkedin.com/company/10634374/>

[https://www.acceptallc.com/signature/youtube.png]<https://www.youtube.com/channel/UC-68TI9KBwCioeSOSln_O7A>

________________________________
